channeldb/payments.go

```diff
@@ -373,6 +374,94 @@ type AttemptInfo struct {
 	Route route.Route
 }
 
+// SentPayment is a wrapper around a sent payment's CreationInfo, AttemptInfo,
```

Comment:
`Sent` implies that it's already been completed, while this payment may still be in flight. I think we can just use `Payment` here perhaps.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2761/commits/ab9b63af0988dbbbd79b8aed0682bc800b723338